### ISSUE TITLE
HC 4.1.x - IE11 problem with pie charts charts 

### ISSUE THREAD

**Issue by sebastianbochan** [highcharts/highcharts#4099]
Let's compare examples:
- http://jsfiddle.net/4occhbmr/ - wrong  - HC 4.1.5 and master branch
- http://jsfiddle.net/4occhbmr/1/  - correct - HC 4.0.4

IE11.0.9600.17691

**Comment by havitoosh** [comment]
Problem still exist on all versions 4.1.x and 4.2.x, even on the latest 4.2.5 version!

**** Please note: the bug is only under Internet Explorer (all versions) but not under Chrome.****

Here is a better example:

- http://jsfiddle.net/4occhbmr/9 - correct! (HC version < 4.1)
- http://jsfiddle.net/4occhbmr/8 - wrong! titles appears with strange gap (HC version >= 4.1)

**Comment by TorsteinHonsi** [comment]
The issue first appears in v4.1.9. The offending commit is b38b949033a26c39f20884fbd17b9bc695fb375a, the fix for #4627.

**Comment by TorsteinHonsi** [comment]
#### Workaround

Disable text shadows:
```js
        series: [{
            type: 'pie',
            name: 'Revenues',
            data: [
                ['כגדיחכגדי חכגד יכגד', 66360.18],
                ['כגכגד כגד כגד כגד כ', 36231.62],
                ['כגכג כגדככג', 23340.17],
                ['כגכגדכגד כגד כגד', 9000],
                ['כגכגד כגד כגדכגד כגד', 6344.44],
                ['כגדכגד', 5394.44],
                ['כגכגדכגד כגד כגדד', 3905.4],
                ['כגכגד כגד כגד כגד432432 כעגד', 2619.83],
                ['כגדכגד כגדכגד כגד', 2209.83],
                ['כגכגדכגד כגדכגדכ כגד כגדד', 2171.77],
                ['כגדכ כגדכגדכ כדג כגדד', 1884.88],
                ['כגכגדכ כגד כגדכגד', 1430.17],
                ['כגדכגד כגדכגדכ גדכגד כגדכגד (543432) כגד', 1014.49],
                ['כגדכגד  כגדכגד כגד כגד', 926.31],
                ['כגדכגד כגדכגד כגדכ גדכגד', 912.07],
                ['כל שאר הלקוחות', 21108.97]
            ],
            dataLabels: {
            	style: {
                	textShadow: 'none'
                }
            }
        }]
```
